src/load/azext_load/tests/latest/test_load.py

```diff
@@ -196,4 +196,6 @@ def test_load_scenarios(self, resource_group, key_vault):
                 '--resource-group {rg} --yes')
         
```

Comment:
Why is this commented out?

Reply:
We'll remove non-required comments at the end. cc:// @Himanshu49